README.md

```diff
@@ -149,6 +152,13 @@ Basic information about the Agent execution are available through the `logs` com
 
 `docker logs dd-agent`
 
+Exec a shell on the container and tail logs for debugging (alpine version shown)
+
+```
+$ docker exec -it dd-agent ash 
+/opt/datadog-agent # tail -f /opt/datadog-agent/logs/dogstatsd.log 
```

Comment:
and also a note saying that logs from the other components of the agents are in the same folder collector.log, forwarder.log and jmxfetch.log. supervisor.log too but we already get this one from `docker logs dd-agent`.